src/const_eval.md

```diff
@@ -84,13 +84,11 @@ slightly) different results. It is advisable to not make array lengths and enum
 discriminants depend on floating point computations.
 
 
-Notable features that const contexts have, but const fn haven't are:
+Notable features that are allowed in const contexts but not in const functions include:
 
 * floating point operations
   * floating point values are treated just like generic parameters without trait bounds beyond
   `Copy`. So you cannot do anything with them but copy/move them around.
-* `dyn Trait` types
-* generic bounds on generic parameters beyond `Sized`
 * comparing raw pointers
 * union field access
```

Comment:
Drive-by fix: unions work just fine nowadays

Reply:
Sounds good, I went ahead and removed that too.